modules/ROOT/nav.adoc

```diff
@@ -3,3 +3,4 @@
 * xref:gov-cloud-features.adoc[Features Included in Government Cloud]
 * xref:gov-cloud-security.adoc[Security in Government Cloud]
 * xref:gov-cloud-account-setup.adoc[Set Up Your Government Cloud Account]
+* xref:runtime-configure-gov.adoc[To Configure the Mule Runtime to Connect to the Government Cloud]
```

Comment:
Minor: For consistency, and ease of maintenance, it would be good to follow the same naming convention for the actual file as previous documents has done. For example, something like: `gov-cloud-runtime-configuration.adoc`

Reply:
Agreed - changed.